mat/eigen.go

```diff
@@ -91,28 +91,30 @@ func (e *EigenSym) Values(dst []float64) []float64 {
 	return dst
 }
 
-// VectorsTo returns the eigenvectors of the decomposition. VectorsTo
-// will panic if the eigenvectors were not computed during the factorization,
-// or if the factorization was not successful.
+// VectorsTo returns the eigenvectors of the decomposition, stored in-place
+// into dst.
 //
-// If dst is not nil, the eigenvectors are stored in-place into dst, and dst
-// must have size n×n and panics otherwise. If dst is nil, a new matrix
-// is allocated and returned.
-func (e *EigenSym) VectorsTo(dst *Dense) *Dense {
+// If dst is empty, VectorsTo will resize dst to be n×n. If dst is
+// non-empty, VectorsTo will panic if dst is not n×n. VectorsTo will also
+// panic if the eigenvectors were not computed during the factorization,
```

Comment:
"When dst is non-empty, VectorsTo will panic if dst is not n×n."

Reply:
Done.